rcp/org.eclipse.rcptt.search/src/org/eclipse/rcptt/search/Q7SearchResultPage.java

```diff
@@ -158,12 +149,6 @@ public StructuredViewer getViewer() {
 		return super.getViewer();
 	}
 
-	@SuppressWarnings("deprecation")
-	private void addDragAdapters(StructuredViewer viewer) {
```

Comment:
Why is this removed?

Reply:
`NavigatorDragAdapter` was removed from `org.eclipse.ui.views.navigator`.

Reply:
How would the page drag and drop element now?

Reply:
Brought it back as internal API.